### DISCUSSION TITLE
Question regardin MPTRJ dataset that doesn't contain magmom

### DISCUSSION THREAD

**Discussion by naveenmohandas** [CederGroupHub/chgnet#216] [General]
Hi, 
 I was checking the MPTRJ dataset and looking the training of the model, I found that there are couple of structures that doesn't contain `magmom` values  (the values are set to `None`). I was wondering, if the structures without magmom would be used for training or would they be ignored during the training when `efsm` is used?

**Comment by bowen-bd** [comment]
The structures with magmom==None will not contribute to magmom loss, but still contribute to energy and force loss terms
